test/TestSetup.sol

```diff
@@ -83,6 +81,7 @@ contract TestSetup is Test, Configured {
         _populateMcFunctionSelectors();
         _populateMa2FunctionSelectors();
         _populateMa3FunctionSelectors();
+        _executeTestTransaction("Execution");
```

Comment:
If each test file is supposed to test a different transaction, then we should:
- name each tx json file a different way (not the generic `Execution`)
- have a constant / immutable / internal function returning the name of the tx json file tested (so we can override it in each child test contract)

Reply:
I think I should separate this Pr in two.
The first PR to build the framework. 
The second to test the rEth listing as example. 
I will close this PR and open to two others if you agree.